Classes/Domain/Repository/IndexRepository.php

```diff
@@ -97,10 +97,11 @@ public function setOverridePageIds($overridePageIds)
      * Select indecies for Backend.
      *
      * @param OptionRequest $options
+     * @param limit values to these pages
      *
      * @return array|\TYPO3\CMS\Extbase\Persistence\QueryResultInterface
      */
-    public function findAllForBackend(OptionRequest $options)
+    public function findAllForBackend(OptionRequest $options, array $allowedPages = [])
```

Comment:
I think `allowedPages`  would better fit into `OptionRequest`

Reply:
I thought options is for the options the user selects in the GUI. But I can change it.

Reply:
mhm, you might be right.
@lochmueller ?

Reply:
Yes... I would merge both ;).... Thanks you @sypets & @okmiim